troposphere/static/js/components/images/detail/versions/Version.react.js

```diff
@@ -120,41 +163,55 @@ export default React.createClass({
                         type={type}
                     />
                 }
-                description={
-                    <div
-                        style={ style.content }
-                    >
-                        <div style={ style.description }
-                            dangerouslySetInnerHTML={{ __html: changeLogHTML }}
-                        />
-                        <div style={ style.availability }
-                        >
-                            { this.renderAvailability() }
-                        </div>
-                    </div>
-                }
+                summary={ this.renderDetail() }
+                detail={ this.renderDetail() }
```

Comment:
As a component, will `summary` and `detail` ever need to have different render functions passed? Or is this a general concept controlled the by the `props`?